### ISSUE TITLE
Windows problems with long filenames

### ISSUE THREAD

**Issue by simsong** [jessek/hashdeep#63]
Converted from [SourceForge issue 2903171](http://sourceforge.net/support/tracker.php?aid=2903171), submitted by jessekornblum

The Win32 versions of the programs have trouble with very long paths. See http://insanecoding.blogspot.com/2007/11/pathmax-simply-isnt.html for background. The problem appears to be the PATH_MAX constant and the getcwd call.

**Comment by IronHand28** [comment]
This issue is ingrained in the OS of the system we are using. It is designed like that but recently Microsoft removed the cap limit of characters. However, if you still are facing this trouble, I highly recommend this tool that I and my team are using - GS RichCopy 360. It is capable of overcoming this restriction and is designed to transfer HUGE amount of files. Read more here - http://www.gurusquad.com/blog/

**Comment by rbeede** [comment]
I've tested with 64-bit builds with the same result of any path > 260 not working.

**Comment by MattAtack** [comment]
Yup - I'm also having this problem with v4.4 and hashdeep64. I tried using Group Policy to enable long filename support, but that didn't help.

Has anyone found a workaround?
